assets/js/components/ParticipantsList.tsx

```diff
@@ -79,30 +70,37 @@ const Participant = ({
   name,
   isModerator,
   isPresenter,
+  moderatorMode,
   eventChannel,
 }: ParticipantProps) => {
   return (
-    <Flex width="15vw" alignItems="center" gap="1">
-      <Avatar name="Dan Abrahmov" src="https://bit.ly/dan-abramov" />
-      <Box p="2">
-        <Heading size="md">{name}</Heading>
-      </Box>
-      {isPresenter && <b>Pres</b>}
-      <Spacer />
+    <div className="Participant">
       {isModerator ? (
+        <img src={crownIcon}></img>
+      ) : isPresenter ? (
+        <img src={starIcon}></img>
+      ) : (
+        <img src={userIcon}></img>
+      )}
```

Comment:
Couldn't we evaluate the source before defining the component? This way we can define `img` element only once.